staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -531,7 +533,7 @@ func (qs *queueSet) removeTimedOutRequestsFromQueueLocked(queue *queue, fsName s
 // Otherwise enqueues and returns true.
 func (qs *queueSet) rejectOrEnqueueLocked(request *request) bool {
 	queue := request.queue
-	curQueueLength := queue.requests.Length()
+	curQueueLength := queue.requests.Width()
```

Comment:
https://github.com/kubernetes/api/blob/9f69feafab6a695158ae715cd035656e9d11b61a/flowcontrol/v1beta1/types.go#L529-L534 still declares a limit on queue _length_ , so comparing that with a sum of seats is not appropriate.

Reply:
Good catch - reverting this line in that PR.